webview/src/plots/components/comparisonTable/ComparisonTableCell.tsx

```diff
@@ -28,15 +31,25 @@ export const ComparisonTableCell: React.FC<ComparisonTableCellProps> = ({
   if (missing) {
     return (
       <div className={styles.noImageContent}>
-        <p>No Plot to Display.</p>
-        <RefreshButton
-          onClick={() =>
-            sendMessage({
-              payload: plot.revision,
-              type: MessageFromWebviewType.REFRESH_REVISION
-            })
-          }
-        />
+        {error ? (
+          <>
+            <ErrorTooltip error={error}>
```

Comment:
Are users going to want to be able to copy the error? I know that when I run into an issue that I can't figure out I need to copy the error message so I can post it somewhere 🤔

Reply:
I will make this copyable in the follow-up where I move the component.